save_translation_file/save_translation_file.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015-TODAY Akretion (<http://www.akretion.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import os
+
+from openerp import models, api, tools
+from openerp.modules import get_module_path
+from openerp.tools.misc import get_iso_codes
+
+
+class IrModuleModule(models.Model):
+    _inherit = 'ir.module.module'
+
+    @api.multi
+    def button_save_translation(self):
+
+        format_ = 'po'
+
+        i18n_path = os.path.join(get_module_path(self.name), 'i18n')
+        if not os.path.isdir(i18n_path):
+            os.mkdir(i18n_path)
+
+        lang_obj = self.env['res.lang']
+        langs = lang_obj.search([('translatable', '=', True)])
+
+        files = [('%s.pot' % self.name, False)]
+        for lang in langs:
+            iso_code = get_iso_codes(lang.code)
+            filename = '%s.%s' % (iso_code, format_)
+            files.append((filename, lang.code))
+
+        for filename, lang in files:
+            path = os.path.join(i18n_path, filename)
+            with open(path, 'w') as buf:
```

Comment:
You should present the dialog to save the file, because not everybody has access to the server (or prefer to make it server-side)